projects/arlas-components/src/lib/components/mapgl-legend/mapgl-legend.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="layer_wrapper">
-  <div class=collection_color matTooltip="Collection: {{collection}}" [style.backgroundColor]="!!collection ? colorService.getColor(collection) : 'transparent'"></div>
+  <div class=collection_color matTooltip="Collection: {{collection | getCollectionDisplayName}}" [style.backgroundColor]="!!collection ? colorService.getColor(collection) : 'transparent'"></div>
```

Comment:
The tooltip should be translated, you could use the translate template, so {{ 'Collection:' | translate:{ collection: collection | getCollectionDisplayName } }}